openapi.json

```diff
@@ -1219,16 +1219,11 @@
                     },
                     "level": {
                         "type": "integer"
-                    },
-                    "updatedOnDate": {
-                        "type": "string",
-                        "format": "date-time"
                     }
                 },
                 "required": [
                     "type",
-                    "level",
```

Comment:
Nepotrebujeme updatedOnDate, ale prečo je odstránený aj level z povinných?

Reply:
on je odstranen i jako field take, takze ted vubec neexistuje

Reply:
ja level vidim, neni to jenom problem zobrazovani?

Reply:
Moja chyba, divný diff.